v2/spanner-common/src/main/java/com/google/cloud/teleport/v2/spanner/migrations/schema/SchemaFileOverride.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright (C) 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package com.google.cloud.teleport.v2.spanner.migrations.schema;
+
+import java.io.Serializable;
+import java.util.Map;
+
+public class SchemaFileOverride implements Serializable {
+
+  private final Map<String, String> renamedTables;
+
+  private final Map<String, Map<String, String>> renamedColumns;
+
+  public SchemaFileOverride(
+      Map<String, String> renamedTables, Map<String, Map<String, String>> renamedColumns) {
+    this.renamedTables = renamedTables;
+    this.renamedColumns = renamedColumns;
+  }
+
+  public Map<String, Map<String, String>> getRenamedColumns() {
```

Comment:
nit - calling this renamed column tuple or such would help distinguish the purpose of this better.

Reply:
Renamed to `getRenamedColumnTupleMap` since it returns a column.